monai/networks/blocks/spatialattention.py

```diff
@@ -46,6 +46,8 @@ def __init__(
         norm_eps: float = 1e-6,
         attention_dtype: Optional[torch.dtype] = None,
         use_flash_attention: bool = False,
+        include_fc: bool = True,
+        use_combined_linear: bool = True,
```

Comment:
Since `SpatialAttentionBlock` is introduced from the `gen-ai` feature branch, I think we don't need to be backward compatible and can set `use_combined_linear` to `False`